endpoints/sample-requests/get-endpoint/invalid/data-corrupted.json

```diff
@@ -0,0 +1,28 @@
+{
+  "description": "Get request doesn't come with a UUID value in the URL query, expect MISSING_KEY error",
+  "config": {
+    "fake_backend": {
+      "stored_data": [
+        {
+          "key": "36-char-uid-maps-to-actual-xml-value",
+          "value": "<HTML>Not prefixed with 'json' nor 'xml' keywords</HTML>"
+        }
+      ]
+    }
+  },
+  "get_request_query": "uuid=",
```

Comment:
No. This test is supposed to throw a `Missing Key` error because the `"get_request_query"`'s value is missing the UUID part, it's incomplete. Maybe in order to make this test clearer I can remove the `"config"` part. Strictly speaking, we don't need infor in the mock backend in order to get the expected result so, the tests passes even if it doesn't come with the mock backend data.

```
 1   {
 2     "description": "Get request doesn't come with a UUID value in the URL query, expect MISSING_KEY error",
 3 -   "config": {
 4 -     "fake_backend": {
 5 -       "stored_data": [
 6 -         {
 7 -           "key": "36-char-uid-maps-to-actual-xml-value",
 8 -           "value": "<HTML>Not prefixed with 'json' nor 'xml' keywords</HTML>"
 9 -         }
10 -       ]
11 -     }
12 -   },
13     "get_request_query": "uuid=",
14     "expected_log_entries": [
15       {
16         "message": "GET /cache: Missing required parameter uuid",
17         "level": 2
18       }
19     ],
20     "expected_metrics": [
21       "RecordGetTotal",
endpoints/sample-requests/get-endpoint/invalid/data-corrupted.json
```

Reply:
Do you want me to modify?

Reply:
Actually it seems like that case is already covered in the test `missing-uuid.json`. Perhaps the point of this test is to throw an error when the stored data value is not prefixed with `json:` or `xml:`?

Reply:
Yes, you are right. I'll modify the `description` field.

Reply:
`endpoints/sample-requests/get-endpoint/invalid/data-corrupted.json` has been updated